cypress/support/commands.ts

```diff
@@ -1,3 +1,16 @@
+
+// Must be declared global to be detected by typescript (allows import/export)
```

Comment:
why did this move to the top of the file?

Reply:
There was a linting error saying Cypress was used before the type was declared.